phabricatoremails/source.py

```diff
@@ -58,7 +68,7 @@ def _request(self, endpoint, **query_parameters):
 
         try:
             # Use POST instead of GET so that the conduit token isn't part of the URL
-            result = requests.post(url, data=body)
+            result = self._session.post(url, data=body)
```

Comment:
Alternatively you could just add the headers to `post` here, without managing a session, but this LGTM.